src/MoBi.Presentation/UICommand/ShowBuildingBlockDiffUICommand.cs

```diff
@@ -8,26 +8,24 @@ namespace MoBi.Presentation.UICommand
    public class ShowBuildingBlockDiffUICommand : IUICommand
    {
       private readonly ISimulationComparisonTask _simulationComparisonTask;
-      private IBuildingBlock _templateBuildingBlock;
-      private IMoBiSimulation _simulation;
+      private IBuildingBlock _simulationBuildingBlock;
+      private readonly ITemplateResolverTask _templateResolverTask;
 
-      public ShowBuildingBlockDiffUICommand(ISimulationComparisonTask simulationComparisonTask)
+      public ShowBuildingBlockDiffUICommand(ISimulationComparisonTask simulationComparisonTask, ITemplateResolverTask templateResolverTask)
       {
+         _templateResolverTask = templateResolverTask;
          _simulationComparisonTask = simulationComparisonTask;
       }
 
-      public ShowBuildingBlockDiffUICommand Initialize(IBuildingBlock templateBuildingBlock, IMoBiSimulation simulation)
+      public ShowBuildingBlockDiffUICommand Initialize(IBuildingBlock simulationBuildingBlock)
```

Comment:
not sure about changing the nomemclature. Something we have everywhere in PKSim and MoBi

Reply:
mind you here it's probably the simulationBuildingBlock so it's fine